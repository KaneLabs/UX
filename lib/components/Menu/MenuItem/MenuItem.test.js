var _interopRequireDefault=require("@babel/runtime/helpers/interopRequireDefault");var _react=_interopRequireDefault(require("react"));var _reactTestRenderer=_interopRequireDefault(require("react-test-renderer"));var _MenuItem=_interopRequireDefault(require("./MenuItem"));var _this=this,_jsxFileName="/Users/kane/Lab/Eros/src/components/Menu/MenuItem/MenuItem.test.tsx";test('MenuItem Renders',function(){var tree=_reactTestRenderer.default.create(_react.default.createElement(_MenuItem.default,{__self:_this,__source:{fileName:_jsxFileName,lineNumber:6,columnNumber:32}})).toJSON();expect(tree).toMatchSnapshot();});