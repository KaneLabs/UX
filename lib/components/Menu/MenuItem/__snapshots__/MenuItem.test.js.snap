// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuItem Renders 1`] = `
<div
  className="css-view-1dbjc4n"
  data-focusable={true}
  data-testid="mb-ripple"
  onClick={[Function]}
  onContextMenu={[Function]}
  onKeyDown={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  style={
    Object {
      "WebkitAlignItems": "center",
      "WebkitBoxAlign": "center",
      "WebkitBoxDirection": "normal",
      "WebkitBoxOrient": "horizontal",
      "WebkitFlexDirection": "row",
      "alignItems": "center",
      "flexDirection": "row",
      "minHeight": "32px",
      "minWidth": "124px",
      "msFlexAlign": "center",
      "msFlexDirection": "row",
      "paddingBottom": "12px",
      "paddingLeft": "24px",
      "paddingRight": "24px",
      "paddingTop": "12px",
      "width": "324px",
    }
  }
  tabIndex="0"
>
  <div
    className="css-text-901oao css-textOneLine-bfa6kz"
    dir="auto"
    onClick={[Function]}
    style={
      Object {
        "color": "rgba(0,0,0,0.87)",
        "fontSize": "16px",
        "fontWeight": "400",
        "letterSpacing": "0.44px",
        "lineHeight": "24px",
        "marginLeft": "0px",
      }
    }
  />
  <div
    className="css-view-1dbjc4n r-pointerEvents-633pao"
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.00)",
        "borderBottomLeftRadius": "0px",
        "borderBottomRightRadius": "0px",
        "borderTopLeftRadius": "0px",
        "borderTopRightRadius": "0px",
        "bottom": "0px",
        "left": "0px",
        "overflowX": "hidden",
        "overflowY": "hidden",
        "position": "absolute",
        "right": "0px",
        "top": "0px",
        "zIndex": 10,
      }
    }
  />
</div>
`;
